ios/Classes/InAppWebView/InAppWebView.swift

```diff
@@ -499,6 +499,10 @@ public class InAppWebView: WKWebView, UIScrollViewDelegate, WKUIDelegate, WKNavi
                 } else if options.cacheEnabled {
                     configuration.websiteDataStore = WKWebsiteDataStore.default()
                 }
+
+                if !options.applicationNameForUserAgent.isEmpty {
+                    configuration.applicationNameForUserAgent = options.applicationNameForUserAgent
+                }

```

Comment:
We also had the similar issue and managed to solve this with a slightly different way. We **append** `applicationNameForUserAgent` instead of overriding it, because we noticed that some tracking parameters in URL link will be missing if we override the default `applicationNameForUserAgent`.

Could you help to check if appending `applicationNameForUserAgent` fixes your issue? If so, I think `appending` would be a safer way to deal with `applicationNameForUserAgent`.